app/controllers/client_api/users/users_controller.rb

```diff
@@ -0,0 +1,164 @@
+module ClientApi
+  module Users
+    class UsersController < ApplicationController
+
+      def current_user_info
+        respond_to do |format|
+          format.json do
+            render template: '/client_api/users/show',
+                  status: :ok,
+                  locals: { user: current_user }
+          end
+        end
+      end
+
+      def change_password
```

Comment:
check this https://github.com/plataformatec/devise/wiki/How-To:-Allow-users-to-edit-their-password

Solution 3.

Reply:
:+1: